specification/migrate/resource-manager/Microsoft.Migrate/stable/2019-10-01/migrate.json

```diff
@@ -2580,9 +2546,6 @@
             }
           }
         },
-        "x-ms-pageable": {
```

Comment:
why are you removing x-ms-pageable from your proxy resource collection GETs?  You can keep this even if you never return a nextLink value.  If your definition doesn't have a "nextLink" property "nextLinkName" can be null. https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable